auth/common.cc

```diff
@@ -113,4 +115,38 @@ ::service::query_state& internal_distributed_query_state() noexcept {
     return qs;
 }
 
+future<> announce_mutations_with_guard(
+        ::service::raft_group0_client& group0_client,
+        const std::vector<mutation>& muts,
+        ::service::group0_guard group0_guard,
+        seastar::abort_source* as) {
+    auto group0_cmd = group0_client.prepare_command(
+        ::service::write_mutations{
+            .mutations{muts.begin(), muts.end()},
+        },
+        group0_guard,
+        "auth: modify internal data"
```

Comment:
Maybe we could put the original query string (without the values, of course) as a description?

On the other hand, maybe it's better to err on the side of vagueness when it comes to auth...

Reply:
I am not exactly sure for what this string is used (debugging?) and how long it can be. Queries can be a bit longer than this current comment.

Reply:
so lets leave it like it is because otherwise I feel we're introducing some another mechanism for tracing. I added trace log inside get_mutations_internal which 'd allow to see auth cql queries when the need be

Reply:
btw yes, you were right about reserve(), it defeats the 2*N capacity growth mechanism